hack/fix_vra_swagger

```diff
@@ -209,6 +204,15 @@ def add_404_not_found(swagger):
             }
 
 
+def add_network_ip_range_return_code(swagger):
```

Comment:
There is a function with same signature. Python will replace the old assignment with this new assignment. If we still want the previous method, we should change the function name here else remove the previous function.